config/common/400-configmap-crio.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: freeze-configmap
+  name: config-freezer
   namespace: knative-serving
 data:
-  runtime-type: containerd
```

Comment:
I delete this because it's useless, with nodeselector, we can have multiple-type runtime, I just set env in deaemonset yaml